### ISSUE TITLE
XBim Xplorer v5.0

### ISSUE THREAD

**Issue by andyward** [xBimTeam/XbimWindowsUI#97]
Uplift and integrate Windows Xplorer with latest v5.0 Essentials & Geometry

- [ ] Uplift to net47 (no netcore for this project)
- [ ] Integrate latest v5 dependencies
- [ ] Create new Pipelines Build
- [ ] Update Nuget packaging
- [ ] Update Plugins (Exchange?)
- [ ] Release to Nuget

**Comment by andyward** [comment]
:shipit: 5.0.118 in Nuget. 

Still needs Plugins and main application packaging with Squirrel

**Comment by CBenghi** [comment]
Hi @andyward,
now that we have the squirrel part done, do you want to go ahead and publish?

**Comment by andyward** [comment]
yes, I think we're good to go on publishing the v5.x Explorer Claudio. Is this something we can set up in Pipelines (as a Release)?

**Comment by CBenghi** [comment]
No, there's a bit of fiddling to do, but I'll be happy to do it.
It's pretty automated with batches on my system.
